### DISCUSSION TITLE
Can parent nodes be disabled, without disabling the child nodes?

### DISCUSSION THREAD

**Discussion by aishwarya2392** [dowjones/react-dropdown-tree-select#478] [Q&A]
If i set disable for a parent node, all the children are also disabled, until i make them explicilty enable. Can this be done through some prop?

**Comment by mrchief** [comment]
No and building something like that would a lot of code and complexity that's unwarranted. Looping through the data object to enable/disable props is far easier even though it may sound counterintuitive.
